tee-worker/identity/client-sdk/packages/client-sdk/src/lib/type-creators/request.ts

```diff
@@ -55,7 +55,7 @@ export async function createRequestType(
       call,
       nonce,
       authentication: api.createType('TCAuthentication', {
-        Web3: signature,
+        [signer.isEmail ? 'Email' : 'Web3']: signature,
```

Comment:
I'm not changing the signature name as it would be a breaking change, for now it would work passing the email verification code as it is also a string

Reply:
Would it be better to use an enum here, take a parameter, and map out the type? What if more types are added in the future?

Reply:
Yes. Well, the proper fix would be change the `createRequestType` signature to properly support `TrustedCallAuthenticated`, but it would be a breaking change and after the demo some types will probably change so we need to refactor this again anyways.